DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: berdie
 Type: Package
 Title: Berdie
-Version: 0.0.2
+Version: 0.0.3
```

Comment:
i feel guilty. hadley has done all the work, so I felt like I should just make a tiny bump